styles/_use-cases.scss

```diff
@@ -543,3 +554,350 @@ html.light {
+    margin-bottom: 20px;
+    padding-bottom: 10px;
+    border-bottom: 1px solid $gray-600;
+  }
+
+  .tokenization-use-case .arrow-button img {
+    width: 15px;
+    height: 15px;
+  }
+
+  .tokenization-stats {
+    width: 100%;
+    height: 250px;
+    border-radius: 8px;
+    background: linear-gradient(88deg, #9A52FF -14.32%, #32E685 45.35%, #19A3FF 100.76%);
+    padding: 4rem 2rem;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    @include media-breakpoint-down(md) {
+      display: block;
+      height: 100%;
+      width: 100%;
+      padding: 0 25%;
+    }
+  }
+
+  .stat-container {
+    color: $black;
+    text-align: center;
+    border-right: 2px solid black;
+
+    @include media-breakpoint-down(md) {
+      border-right: none;
+      padding-bottom: 3rem;
+      padding-top: 2rem;
+      border-bottom: 2px solid black;
+    }
+    
+  }
+
+  .stat-container:last-child {
+    border: none;
+  }
+
+  .stat-container .stat {
+    font-size: 3rem;
+    font-weight: 300;
+  }
+
+  .stat-container p {
+    font-weight: 400;
+  }
+
+  /* Work-around for border gradient and radius */
+  .security-card {
+    position: relative;
+    border-radius: 0.5rem;
+    background-color: transparent;
+    white-space: normal;
+    box-sizing: border-box;
+  }
+
+  .security-card::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    border-radius: inherit;
+    padding: 1px;
+    background: linear-gradient(90deg, #d91aff 26.41%, #1aa4ff 100.32%);
+    mask: 
+      linear-gradient(#fff 0 0) content-box, 
+      linear-gradient(#fff 0 0);
+    mask-composite: exclude;
+    -webkit-mask-composite: xor;
+    z-index: -1;
+  }
+
+  .security-card p {
+    margin-bottom: 0 !important;
+  }
+
+  .security-card .h6 {
+    @include media-breakpoint-down(sm) {
+      font-size: 1.25rem;
+    }
+  }
+
+  .security-card-grid {
+    gap: 1rem;
+    grid-template-columns: repeat(2, 1fr);
+
+    @media (min-width: 1200px) {
+      grid-template-columns: repeat(4, 1fr);
+    }
+
+    @media (max-width: 768px) {
+      grid-template-columns: repeat(1, 1fr);
+    }
+  }
+
+  .video-external-link .link-text {
+    margin-left: 0.25rem;
+  }
+
+  .video-external-link {
+    margin-bottom: 9px;
+  }
+
+  .tokenization-color-bar {
+    align-self: stretch;
+    height: 0.25rem;
+    border-radius: 2rem;
+    background: var(--Gradient-3, linear-gradient(90deg, #FEFF01 0%, #FF2D9A 30.82%, #E24CFF 64.01%, #9A52FF 100%));
+  }
+
+  .project-cards-container {
+    gap: 3rem;
+  }
+
+  .project-cards {
+    width: 100%;
+  }
+
+  .project-cards .project-name {
+    word-break: break-word;
+  }
+
+  .project-cards .col {
+    padding: 32px;
+    background-color: $black;
+    border-bottom-left-radius: 0.5rem;
+    border-bottom-right-radius: 0.5rem;
+    border-top-left-radius: 0.25rem;
+    border-top-right-radius: 0.25rem;
+    text-align: center;
+    position: relative;
+  }
```

Comment:
The height of the cards can vary depending on whether the project name gets wrapped onto a second line, causing the control arrows below them to move up and down as you're clicking through the carousel. This can be fixed by adding a minimum height to the cards.

If you've added the `card` class to these elements already (see my suggestion on line 204 of `tokenization/index.page.tsx`) then that already sets minimum height on the cards, but it's too large. Also, the rest of these styles become unnecessary because they're part of the `card` class. So instead, all we need to do is set a smaller minimum height:

```suggestion
  .project-cards .card {
    min-height: 240px;
  }
```

Reply:
![Screenshot 2024-06-12 at 2 41 09 PM](https://github.com/XRPLF/xrpl-dev-portal/assets/71317875/c44c4ac0-91d3-41ce-821a-7eacd680eb40)
240px would leave empty spaces on mobile screen with the reduced font. Maybe we should reduce min-height to 200px on small screen?

Reply:
Less than ~235px causes it to change heights in some cases, based on my testing